addons/purchase/views/purchase_views.xml

```diff
@@ -460,7 +460,7 @@
                         domain="[('activity_exception_decoration', '!=', False)]"/>
                     <group expand="0" string="Group By">
                         <filter string="Vendor" name="vendor" domain="[]" context="{'group_by': 'partner_id'}"/>
-                        <filter string="Purchase Representative" name="representative" domain="[]" context="{'group_by': 'user_id'}"/>
+                        <filter string="Buyer" name="buyer" domain="[]" context="{'group_by': 'user_id'}"/>
```

Comment:
I would leave the `name="representative"` as it's rather technical and could break filters or default values